### ISSUE TITLE
[Bug Report] Gemma-2b model fails when splitting QKV input

### ISSUE THREAD

**Issue by Aaquib111** [neelnanda-io/TransformerLens#558]
If you are submitting a bug report, please fill in the following details and use the tag [bug].

**Describe the bug**
Running gemma-2b on transformer lens fails when `set_use_split_qkv_input` is set to true.

**Code example**
A minimal example:
```python
model_name = "google/gemma-2b"
tokenizer = AutoTokenizer.from_pretrained(model_name)
model = HookedTransformer.from_pretrained(
    model_name,
    tokenizer=tokenizer,
    device='cuda',
    default_padding_side="left",
    fold_ln=False,
    fold_value_biases=False,
    center_writing_weights=False,
)

text = '''
TransformerLens lets you load in 50+ different open source language models,
and exposes the internal activations of the model to you. You can cache
any internal activation in the model, and add in functions to edit, remove
or replace these activations as the model runs.
'''
input_ids = torch.tensor(tokenizer.encode(text)).unsqueeze(0).cuda()
print('Running without splitting QKV')
print(model(input_ids)) # Does not fail!

print('Running with splitting QKV')
model.set_use_split_qkv_input(True)
print(model(input_ids)) # Fails now!
```

Error:
```
in GroupedQueryAttention.calculate_attention_scores(self, q, k)
   [1135] """Calculate attention scores from Q and the unexpanded K matrix.
   [1136] K will be expaned from [batch, pos, n_key_value_head, d_head] to [batch, pos, n_query_heads, d_head] using torch.repeat_interleave.
   [1137] 
   (...)
   [1143]     Float[torch.Tensor, "batch head_index query_pos key_pos"]: The attention scores.
   [1144] """
   [1145] k = torch.repeat_interleave(k, dim=2, repeats=self.repeat_kv_heads)
-> [1146] return super().calculate_attention_scores(q, k)

in AbstractAttention.calculate_attention_scores(self, q, k)
    [675] def calculate_attention_scores(
    [676]     self,
    [677      q: Float[torch.Tensor, "batch query_pos head_index d_head"],
    [678]     k: Float[torch.Tensor, "batch key_pos head_index d_head"],
    [679] ) -> Float[torch.Tensor, "batch head_index query_pos key_pos"]:
    [680]      attn_scores = (
--> [681]         einsum(
    [682]            "batch query_pos head_index d_head, \
    [683]                 batch key_pos head_index d_head \
    [684]                 -> batch head_index query_pos key_pos",
    [685]             q,
    [686]             k,
    [687]        )
    [688] self.attn_scale
    [689]     )
    [690]     return attn_scores

RuntimeError: einsum(): subscript h has size 64 for operand 1 which does not broadcast with previously seen size 8

```
**System Info**
 * GPU: A6000 
 * TransformerLens installed via `pip install transformer_lens`
 * OS: Linux
 * Python version: Python 3.10.13

### Checklist

- [x] I have checked that there is no similar [issue](https://github.com/neelnanda-io/TransformerLens/issues) in the repo (**required**)

**Comment by wesg52** [comment]
I believe this should be fixed by #520 but the change isn't in the latest release.

**Comment by ArthurConmy** [comment]
Thanks @wesg52, I just triggered the release of v1.16.0: https://github.com/neelnanda-io/TransformerLens/releases/tag/v1.16.0 which should land in ~10 minutes and fix this problem. For now @Aaquib111 you can pip install from the github repo.
